### DISCUSSION TITLE
Meeting Notes

### DISCUSSION THREAD

**Discussion by LauraNWRM** [NewGraphEnvironment/restoration_framework#17] [General]
Is this an appropriate way to share meeting notes?

**Comment by NewGraphEnvironment** [comment]
see note made here https://github.com/NewGraphEnvironment/restoration_framework/discussions/18 .  Meeting notes can be kept somewhere specific to the restoration_wedzin_kwa_2024 project that everyone on the team has access to and could potentially be the type of thing that could be considered best kept private .  

Google drive is also a great place to store project specific data as well and there might be other systems already in place among the team that don't require any additional setup
